docs/user-guide/work-packages/work-package-table-configuration/README.md

```diff
@@ -2,27 +2,30 @@
 sidebar_navigation:
   title: Configure work package table
   priority: 700
-description: How to configure the work package table in OpenProject?
+description: How to configure the work package list in OpenProject?
 robots: index, follow
-keywords: work packages table configuration
+keywords: work packages table configuration, work package list, columns, filter, group
 ---
 
 # Work package table configuration
 
-| Feature                                                      | Documentation for                                            |
+| Topic                                                        | Content                                                      |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | [Add or remove columns](#add-or-remove-columns-in-the-work-package-table) | How to add or remove columns in the work package table?      |
 | [Filter work packages](#filter-work-packages)                | How to filter in the work package list?                      |
 | [Sort the work package list](#sort-the-work-package-list)    | How to sort within the work package list?                    |
-| [Display settings](#how-to-switch-from-flat-list-to-hierarchy-mode?) | How to switch from flat list to hierarchy mode in the work package list? |
+| [Display settings](#flat-list,-hierarchy-mode-and-group-by)  | Get to know the flat list, the hierarchy mode and the group by feature. |
+| [Aggregation by project](#aggregation-by-project)            | How to display an aggregated view of all milestones of multiple projects? |
 | [Attribute highlighting (Premium Feature)](#attribute-highlighting-premium-feature) | How to highlight certain attributes in the work package list? |
-| [Save work package views](#save-work-package-views)          | How to save a new work package view?                         |
+| [Save work package views](#save-work-package-views)          | How to save a new work package view and how to change existing ones? |
 
 You can configure the work package table view in OpenProject to display the information that you need in the list.
 
-You can change the header in the table and add or remove columns, filter and group work packages or sort them according to a specific criteria. Save the view to have it available directly from your project menu.
+You can change the header in the table and add or remove columns, filter and group work packages or sort them according to a specific criteria. Also, you can change between a flat list view, a hierarchy view and a grouped view.
```

Comment:
We can change "sort them according to a specific criteria" to "sort them according to specific criteria"